docs/api-server-walkthrough.md

```diff
@@ -95,11 +100,19 @@ And some of ours:
 }
 ```
 
+kubectl needs a basic kubeconfig setup. A kubeconfig consists of three
+sections that can be set with the following three commands:
+
+1. `kubectl config set-credentials service-catalog-creds --username=admin --password=admin`
+1. `kubectl config set-cluster service-catalog-cluster --server=https://localhost:6443 --certificate-authority=/var/run/kubernetes/apiserver.crt`
+1. `kubectl config set-context service-catalog-ctx --cluster=service-catalog-cluster --user=service-catalog-creds`
+1. `kubectl config use-context service-catalog-ctx`
```

Comment:
possibly create a `make` target, as a follow up, for these commands? @pmorie also suggested using `direnv` to achieve the same